src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -301,6 +301,20 @@ void WriteLog<I>::load_existing_entries(pwl::DeferredContexts &later) {
   }
   this->update_sync_points(missing_sync_points, sync_point_entries, later,
                            MIN_WRITE_ALLOC_SSD_SIZE);
+  if (m_first_valid_entry > m_first_free_entry) {
+    m_bytes_allocated = this->m_log_pool_size - m_first_valid_entry  + m_first_free_entry - DATA_RING_BUFFER_OFFSET;
+  } else {
+    m_bytes_allocated = m_first_free_entry - m_first_valid_entry;
+  }
+}
+
+ // For SSD we don't calc m_bytes_allocated in this
+template <typename I>
+void WriteLog<I>::calc_allocated_and_cached_data(std::shared_ptr<pwl::GenericLogEntry> log_entry, uint32_t alloc_size) {
+  auto gen_write_entry = static_pointer_cast<GenericWriteLogEntry>(log_entry);
+  if (log_entry->is_write_entry()) {
+    this->m_bytes_cached += log_entry->write_bytes();
```

Comment:
I'm not sure if `m_bytes_cached` is useful in `ssd` cache mode. You may want to verify that.